### DISCUSSION TITLE
Create UI for Lecture 12 Stable coin

### DISCUSSION THREAD

**Discussion by touqeerShah** [Cyfrin/foundry-full-course-cu#158] [Ideas]
https://www.youtube.com/watch?v=r1_Vyo5o-nA

I create UI in nextJS for Stable Coin Project which help other to understand more easily working and flow of project and Please look at I try to push code but I have no permission for it

**Comment by ChefAharoni** [comment]
You can fork this repo, edit whatever you want in the fork, and then suggest a pull request to the main repo.

**Reply by cromewar** [reply]
Yes please @touqeerShah create a pull request and I will add this to the repo as reference.

**Reply by touqeerShah** [reply]
yes Already created pull request for it
https://github.com/Cyfrin/foundry-defi-stablecoin-f23/pull/17

**Reply by PatrickAlphaC** [reply]
Can you make a pull request on this repo? (not the defi-stablecoins repo)?

**Reply by touqeerShah** [reply]
I made I pull request but you said no NextJS UI, So it's ok thank you
